CONTRIBUTING.md

```diff
@@ -12,9 +12,54 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-#Adding a new section
+## Permissions & Contributions
 
-##Structure/Pattern
+There are two ways in which you can help update the content:
+
+* **One-off:** \
+If you are not a regular contributor to the project, but you would like to contribute some changes, the best way to do it is:
+
+  1. Fork the repo
+  1. Create a PR
+
+* **Periodic and regular contributions:** \
+If you plan to update the content semi-regularly or regularly, you can be added to the project's Contributors group. Please contact one of the project's administrators to be added to the group.
+
+## Git guidance
+
+Consistent with the practices suggested in this playbook, please follow the specifics regarding git as described in this section.
+
+### Branch naming convention
+
+In this repo, we will the following branch naming conventions:
+
+| Branch Type       | Pattern     | Example     |
+| ------------- | ---------- | ----------- |
+|  Feature | feature/\<issue#>-\<short description> | feature/498-reorganize-scm-section |
+| Bug Fix | fix/\<bug#>-\<short description> | bug/978-correct-grammar-myfile.md |
+| Personal | users/\<alias>/\<short description> | johndo/rewrite-grooming-section
+
+> **Note:**
+>
+> * Use of personal branches should be minimized since most work should refer back to a feature/bug fix in the backlog.
+> * Mind the capitalization of the branch prefix (feature, fix, users). Tools that diplay branches as a hierarchy are typically case sensitive, and will display different hierarchies for the same words with different capitalization.
+
+### Linting
+
+Please install the [markdownlint extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) and ensure that all rules are followed. This will help ensure consistency in the look and feel of the documentation in this repo.
+
+### Merging strategy
```

Comment:
Since we haven't decided on a merging strategy, either decide on one, (I prefer non-linear), or remove the section, or add a TODO to decide. As is, this section doesn't add anything to the document

Reply:
Correction: Looks like the default is squash and commit, and that most people so far has done this so we should probably change this to linear

Reply:
I have added the task in the backlog - I'll change appropriately.